sql/table.sql

```diff
@@ -256,6 +256,29 @@ create table variation_synonym (
   key source_idx (source_id)
 );
 
+/**
+@table allele_synonym
+
+@colour #B22222
+@desc This table allows for the allele of a variant to have multiple IDs.
+
+@column allele_synonym_id     Primary key, internal identifier.
+@column variation_id          Foreign key references to the @link variation table.
+@column hgvs                  HGVS of variant e.g. NC_000001.11:g.230710048A>G
+@column name                  Name of the allele synonym e.g. CA127784
+
+@see variation
+*/
+
+CREATE TABLE allele_synonym (
+  allele_synonym_id int(10) unsigned NOT NULL AUTO_INCREMENT,
+  variation_id      int(10) unsigned NOT NULL,
+  hgvs              varchar(600) NOT NULL,
```

Comment:
We refer to this value as hgvs_genomic in the transcript_variation table; it may be useful to be consistent.

Reply:
Is the only genomic hgvs going to be used?

Reply:
I can't currently think of a reason to use any other.

Reply:
I'll change the name. The ClinGen Allele Registry supports search using HGVS expressions for a transcripts, genomic and protein sequences .